### PR TITLE
Fix test invocations in readme; delete outdated lamp readme

### PR DESCRIPTION
**Pull request by vrachev**
Fixing some things I encountered during onboarding

### CODE REVIEW

File: src/lamplib/README.md [hunk 1]
```diff
@@ -1,110 +0,0 @@
...
-```
-
-Lint most of the YAML files in the Genny repository,
-including workloads, phases, and `evergreen.yml`.
-
-Make you run the script from the root of the Genny repository
-(so it can pick up on the `.yamllint` linter config file)
-
-
-Script: `genny-auto-tasks`
```
Comment (vrachev) [inline src/lamplib/README.md:52]: This is the only section that isn't reflected in the general readme. Is this something we should port over? Not sure if anyone is actually running genny-auto-tasks locally.

Comment (rtimmons) [reply src/lamplib/README.md:52]: We can kill it. Auto-tasks is a part of `run-genny`.

**Review by rtimmons — APPROVED** [review]

**Comment by vrachev** [comment]
evergreen merge

**Comment by vrachev** [comment]
evergreen merge

**Pull request merged**